cmd/mount_zfs/Makefile.am

```diff
@@ -19,4 +19,5 @@ mount_zfs_LDADD = \
 	$(top_builddir)/lib/libuutil/libuutil.la \
 	$(top_builddir)/lib/libzpool/libzpool.la \
 	$(top_builddir)/lib/libzfs/libzfs.la \
-	$(top_builddir)/lib/libzfs_core/libzfs_core.la
+	$(top_builddir)/lib/libzfs_core/libzfs_core.la \
+	$(TIRPC_LIBS)
```

Comment:
Rather than placing this in every consumer of libspl/libzfs, this should go into `libspl_la_LIBADD` in lib/libspl/Makefile.am.